app/app.js

```diff
@@ -4,13 +4,35 @@ var GameUpApp = angular.module('GameUpApp', ['ngRoute', 'ngMessages', 'parse-ang
 
 
 GameUpApp.run(['$rootScope', function($scope) {
+
   $scope.currentUser = Parse.User.current();
 
+  $scope.alerts = {};
+  $scope.user = {
+    name: Parse.User.current()==null ? "Guest" : Parse.User.current().getUsername(),
+    email: Parse.User.current().getEmail() == null ? "johndoe@example.com" : Parse.User.current().getEmail(),
```

Comment:
This will produce errors if Parse.User.current() is null. We should do one if statement check for this before setting $scope.user.